plugins/obs-filters/data/mask_background_filter.effect

```diff
@@ -0,0 +1,64 @@
+uniform float4x4 ViewProj;
```

Comment:
This file should not be part of this PR. It is not relevant to the fixed problem, and is a feature addition.